CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog
 
+### v6.0.0 Apr 22st, 2024
```

Comment:
Let's just put "Apr 22" here. "Apr 22st" is not correct (same applies to line 7 - can you update this as well?)